src/hardware/sensors/config/ellipse_N.yaml

```diff
@@ -20,11 +20,11 @@
       # Enable ROS odometry messages.
       enable: false
       # Publish odometry transforms.
-      publishTf: true
+      publishTf: false
```

Comment:
no transforms for ellipse?

Reply:
ye, we've got Robot Localization for that now (taking in SBG + more odom)